CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
- # v0.9.10
+# v0.9.11
```

Comment:
I'd almost think about deleting this CHANGELOG (thought I had in a previous commit?) and use the releases section as the source of truth, e.g.:

https://github.com/Sifchain/sifnode/releases/tag/betanet-0.9.10

Reply:
This sounds good. I recently saw you had a PR for that outstanding which I approved, will pick it up later. https://github.com/Sifchain/sifnode/pull/2014